packages/component-library/stories/CardList.story.js

```diff
@@ -4,7 +4,21 @@ import { storiesOf } from "@storybook/react";
 import { checkA11y } from "@storybook/addon-a11y";
 import { CardList } from "../src/index";
 import { storybookStyles } from "./storyStyles";
-import { SampleCard, SampleCardB } from "./CivicCardSample";
+import {
+  FilterSampleCard,
+  FilterSampleCardB,
+  FilterSampleCardC,
+  FilterSampleCardD,
+  FilterSampleCardE,
+  FilterSampleCardF,
+  FilterSampleCardG,
+  FilterSampleCardH,
+  FilterSampleCardJ,
+  FilterSampleCardK,
+  FilterSampleCardL,
+  FilterSampleCardM,
+  FilterSampleCardN
+} from "./CivicCardSampleFilter";
```

Comment:
It would be better practice here to dynamically generate these cards since they're almost identical and then dynamically generate the `CardRegistryMock`. As much as possible we want to try and avoid writing code more than once. A good rule of thumb is 1-2 repeats, but it depends on the nature of the code. This, however, is a great candidate for a component generator function!